darts/tests/utils/tabularization/test_create_lagged_training_data.py

```diff
@@ -101,6 +102,9 @@ def get_feature_times(
                 future, lags_future
             )
             times = times.intersection(future_times)
+        # Apply stride
+        if stride > 1:
```

Comment:
as mentioned in tabularization.py, this should ideally take from the end of the series